### PR TITLE
refactor(LedgerAccounts): code

### PR DESCRIPTION
**Pull request by Crazy-MIXER**
- Move text to i18n
- Change the imports to absolute
- Change the color on `Color`
- Replace the `textAligh ` props  
- Replace navigation hooks
- Migration the `nextScreen` prop from the `ledger `screen to the `ledger-account` screen

### CODE REVIEW

File: src/screens/ledger-accounts.tsx [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
 
   const onPressAdd = useCallback(
     (address: string) => {
       navigation.navigate('ledgerVerify', {
+        nextScreen: nextScreen,
         address,
-        deviceId: route.params.deviceId,
-        deviceName: route.params.deviceName,
+        deviceId: deviceId,
+        deviceName: deviceName,
```
Comment (gHashTag) [inline src/screens/ledger-accounts.tsx:20]:
```js
{
         nextScreen,
         address,
         deviceId,
         deviceName,
}
```

File: src/components/ledger-accounts/ledger-accounts.tsx [hunk 2]
```diff
@@ -24,14 +23,14 @@ export const LedgerAccounts = ({deviceId, onAdd}: LedgerDeviceProps) => {
     const iter = runUntil(deviceId, eth => eth.getAddress(ETH_HD_PATH, false));
     requestAnimationFrame(async () => {
       let done = false;
+      let address: string;
       do {
         const resp = await iter.next();
         done = resp.done;
+        address = resp.value.address;
```
Comment (gHashTag) [inline src/components/ledger-accounts/ledger-accounts.tsx:30]: `const address = resp.value.address;`

File: src/components/ledger-accounts/ledger-accounts.tsx [hunk 3]
```diff
@@ -24,14 +23,14 @@ export const LedgerAccounts = ({deviceId, onAdd}: LedgerDeviceProps) => {
     const iter = runUntil(deviceId, eth => eth.getAddress(ETH_HD_PATH, false));
     requestAnimationFrame(async () => {
       let done = false;
+      let address: string;
```
Comment (gHashTag) [inline src/components/ledger-accounts/ledger-accounts.tsx:26]: remove

**Review by gHashTag — CHANGES_REQUESTED** [review]
Fix comments

File: src/components/ledger-accounts/ledger-accounts.tsx (hunk 3 above)
Comment (Crazy-MIXER) [reply src/components/ledger-accounts/ledger-accounts.tsx]: fix all

**Review by gHashTag — APPROVED** [review]

File: src/screens/ledger.tsx [hunk 4]
```diff
@@ -60,11 +57,6 @@ export const LedgerScreen = () => {
         options={{
           title: 'Verify',
         }}
-        initialParams={{
```
Comment (vivalaakam) [inline src/screens/ledger.tsx:63]: why did you remove these?

Comment (Crazy-MIXER) [reply src/screens/ledger.tsx:63]: image

Comment (vivalaakam) [reply src/screens/ledger.tsx:63]: this screen is run for two different flows and this prop determines which screen is next.

Comment (Crazy-MIXER) [reply src/screens/ledger.tsx:63]: Ok I'll fix it, but then what should be `nextScreen ` in the ledger-accounts? It wasn't here before

**Review by vivalaakam — CHANGES_REQUESTED** [review]
fix conflicts

File: src/screens/ledger.tsx (hunk 4 above)
Comment (Crazy-MIXER) [reply src/screens/ledger.tsx]: I try fix it

**Review by vivalaakam — APPROVED** [review]
